src/placeos-rest-api/controllers/repositories.cr

```diff
@@ -212,5 +212,42 @@ module PlaceOS::Api
         end
       end
     end
+
+    get "/:id/releases", :releases do
+      releases = Api::Repositories.releases(
+        repository: current_repo,
+        request_id: request_id,
+      )
+
+      render json: releases
+    end
+
+    def self.releases(repository : Model::Repository, request_id : String)
+      case repository.repo_type
+      in .interface?
+        # Dial the frontends service
+        FrontendLoader::Client.client(request_id: request_id) do |frontends_client|
+          frontends_client.releases(repository.folder_name)
+        end
+      in .driver?
+        Log.info { {
+          message:       "can only get releases for interface repositories",
+          repository_id: repository.id,
+          folder_name:   repository.folder_name,
+          name:          repository.name,
+          type:          repository.repo_type.to_s,
+        } }
+      end.tap do |result|
+        if result.nil?
+          Log.info { {
+            message:       "failed to retrieve releases",
+            repository_id: repository.id,
+            folder_name:   repository.folder_name,
+            name:          repository.name,
+            type:          repository.repo_type.to_s,
+          } }
+        end
```

Comment:
I'd check that the repository is an interface before calling this method, and return a `400` if it isn't